gui/default/syncthing/core/syncthingController.js

```diff
@@ -1097,6 +1097,126 @@ angular.module('syncthing.core')
             }
         };
 
+        $scope.deviceStatusIcon = function(cfg) {
+            switch ($scope.deviceStatus(cfg)) {
+                case 'insync':
+                    return 'fa-check';
+                case 'unused-insync':
+                    return 'fa-unlink';
+                case 'syncing':
+                    return 'fa-sync';
+                case 'paused':
+                    return 'fa-pause';
+                case 'unused-paused':
+                    return 'fa-unlink';
+                case 'disconnected':
+                    return 'fa-power-off';
+                case 'disconnected-inactive':
+                    return 'fa-power-off';
+                case 'unused-disconnected':
+                    return 'fa-unlink';
+            }
+        };
+
+        $scope.deviceStatusText = function(device) {
+            switch ($scope.deviceStatus(device)) {
+                case 'insync':
+                    return $translate.instant('Up to Date');
+                case 'unused-insync':
+                    return $translate.instant('Connected (Unused)');
+                case 'syncing':
+                    return $translate.instant('Syncing') + ' (' + percentFilter($scope.completion[device.deviceID]._total) + ', ' + binaryFilter($scope.completion[device.deviceID]._needBytes) + 'B)';
```

Comment:
I think the multi-line variant in the other function is easier to read.  Split here as well please?

Reply:
Done for all these combined texts. Not sure if it's too much or not, but I followed the formatting recommendation from https://google.github.io/styleguide/javascriptguide.xml?showone=Multiline_string_literals#Multiline_string_literals.